tests/end_to_end/PlainOpcodes/conftest.py

```diff
@@ -1,7 +1,7 @@
 import pytest_asyncio
 
 
-@pytest_asyncio.fixture(scope="package")
+@pytest_asyncio.fixture
```

Comment:
this is going to redeploy a new counter each time, why did you remove this? (same for all contracts)

Reply:
otherwise this wouldn't run the tests at all. Would you have time to TAL ?